README.md

```diff
@@ -26,6 +27,8 @@ Additional options:
 * `-d` Install TMP dir. Default: "/tmp/debian11-install"
 * `-v` Verbose on or off. Default: "on"
 * `-b` Git branch to clone. Default: "main"
+* `-h` Add advanced hardening or not (yes/no). Default: "no"
```

Comment:
why do we need two switches for advanced hardening?
-n "hardening" and -h

proposing -n "hardening" and one new profile like -n "full+hardening"

Reply:
-n "hardening"  - it's just hardening and nothing more. If you want to install app+hardening - it's necessary to use -h "yes"